CONTRIBUTING.md

```diff
@@ -1310,3 +1311,22 @@ your PR branch. If you happen to have write access to this repo, you can choose
 to use `ghstack` to eliminate this nondeterminism for GitHub Actions jobs on
 your PRs, but it will still be present for the select CircleCI jobs listed
 above.
+
+## Dev Infra Office Hours
+The Pytorch Dev Infra team is hosting office hours every [Friday at 10AM PST](https://fb.zoom.us/j/91990997296?pwd=b1V1NXVzMGtiNmx4TzJUUEZ2MTdsdz09). We are primarily responsible for a few initiatives including:
+
```

Comment:
Instead of copying and pasting the whole contents of the wiki, add a link to the wiki itself here and summarize the contents into a single sentence.